SPECS/rabbitmq-server/generate-rabbitmq-server-tarball.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Not a blocker:
Can we please add some help here so that it becomes trivial to run this for any other member :)

Reply:
Help added, additional notes for recreation are in the spec file.